.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -157,7 +157,9 @@ jobs:
           ./emsdk/emsdk install 2.0.14
           ./emsdk/emsdk activate 2.0.14
       - name: "Install Dependencies"
-        run: sudo apt-get update && sudo apt-get install -y ninja-build clang-format-12
+        run: | 
+          sudo apt-get update && sudo apt-get install -y ninja-build
+          pip install --user --break-system-packages clang-format==18.1.8
```

Comment:
can you add the lines so we ensure that `clang-format` is in the right place
```
          test -x ~/.local/bin/clang-format
          test -x /home/runner/.local/bin/clang-format
```

Reply:
Done